offchain-modules/cli/src/commands/mod.rs

```diff
@@ -24,33 +27,69 @@ pub fn handler(opt: Opts) -> Result<()> {
         // verify ckb spv proof && unlock erc20 token.
         SubCommand::Unlock(args) => unlock_handler(args),
         SubCommand::TransferFromCkb(args) => transfer_from_ckb_handler(args),
-
-        SubCommand::EthRelay(args) => eth_relay_handler(args),
-        SubCommand::CkbRelay(args) => ckb_relay_handler(args),
     }
 }
 
 pub fn approve_handler(args: ApproveArgs) -> Result<()> {
     println!("approve_handler args: {:?}", &args);
-    let from: H160 = H160::from_slice(args.from.as_ref());
-    let to = H160::from_slice(args.to.as_ref());
-    let hash = approve(from, to, args.rpc_url, args.chain_id, args.private_key_path);
-    log::info!("approve tx_hash: {:?}", &hash);
+    let from: H160 = H160::from_slice(hex::decode(args.from).unwrap().as_slice());
+    let to = H160::from_slice(hex::decode(args.to).unwrap().as_slice());
+    let hash = approve(from, to, args.rpc_url, args.private_key_path);
+    println!("approve tx_hash: {:?}", &hash);
     Ok(())
 }
 
-pub fn lock_handler(args: LockArgs) -> Result<()> {
+pub fn lock_token_handler(args: LockTokenArgs) -> Result<()> {
     println!("lock_handler args: {:?}", &args);
```

Comment:
All the debug message should use `log::Debug!()`.
The output of the command can be shown by `println!`.